instance/models/openedx_instance.py

```diff
@@ -160,10 +167,41 @@ def spawn_appserver(self):
             # Copy the current value of each setting into the AppServer, preserving it permanently:
             configuration_database_settings=self.get_database_settings(),
             configuration_storage_settings=self.get_storage_settings(),
+            configuration_extra_settings=self.get_extra_settings(),
             **instance_config
         )
         if app_server.provision():
             # If the AppServer provisioned successfully, make it the active one:
             # Note: if I call spawn_appserver() twice, and the second one provisions sooner, the first one may then
             # finish and replace the second as the active server. We are not really worried about that for now.
             self.set_appserver_active(app_server.pk)
+
+
+class OpenEdXProductionInstance(OpenEdXInstance):
+    """
+    OpenEdXProductionInstance: An OpenEdXInstance using production settings
+    """
+
+    LATEST_RELEASE = "named-release/dogwood.rc"
+
+    def save(self, **kwargs):
```

Comment:
Also - sorry if I sound like a broken record on this, but I thought we agreed to only have different options/attributes on the Production instance, but no difference in logic. Adding methods to this class modifies the logic, even if it's to set default values. @bradenmacdonald 's suggestions already address how to move most of the logic on the parent class, though - just a note to make sure this class doesn't include methods at the end : )